ainow/templates/ainow/_nav.html

```diff
@@ -50,4 +53,11 @@
 <li><a href="{% url 'sponsorship' %}">Sponsorship</a></li>
 <li><a href="{% url 'press' %}">Press Enquiries</a></li>
 
+{% elif schedule.slug == '2020' %}
+
+<li><a href="{% url 'schedule2020summary' %}">Summary</a></li>
+<li><a href="{% url 'sponsorship' %}">Sponsorship</a></li>
+<li><a href="{% url 'press' %}">Press Enquiries</a></li>
+<li><a href="https://www.mysociety.org/tag/tictec/">Blog</a></li>
+
```

Comment:
The top nav looks a bit broken when the screen is narrower than about 990px but before the mobile breakpoint kicks in:

![image](https://user-images.githubusercontent.com/22996/64539450-6cd2b400-d316-11e9-94ce-be0641a0b462.png)

Reply:
This is an annoying one, mainly due to the number of pages we're linking to - I wonder how we'd feel about just ignoring this for now, as once more details are finalised and the date moves closer the navigation will change again

Reply:
Rather than adding CSS for this - seems edge-casey